pkg/registry/batch/rest/storage_batch.go

```diff
@@ -36,6 +37,10 @@ func (p RESTStorageProvider) NewRESTStorage(apiResourceConfigSource genericapise
 	if apiResourceConfigSource.AnyResourcesForVersionEnabled(batchapiv2alpha1.SchemeGroupVersion) {
 		apiGroupInfo.VersionedResourcesStorageMap[batchapiv2alpha1.SchemeGroupVersion.Version] = p.v2alpha1Storage(apiResourceConfigSource, restOptionsGetter)
 		apiGroupInfo.GroupMeta.GroupVersion = batchapiv2alpha1.SchemeGroupVersion
+		apiGroupInfo.SubresourceGroupVersionKind = map[string]unversioned.GroupVersionKind{
```

Comment:
This is telling api server that GVK ScheduledJobs is backed by GVR ScheduledJobs. Without it the API server won't be even able to start complaining about one operation supporting two distinct paths.